packages/enclave/src/v3/eth/wallets/wallets.service.ts

```diff
@@ -1,4 +1,160 @@
 import { Injectable } from "@nestjs/common";
+import { SDK, Transaction } from "@haechi-labs/henesis-wallet-core";
+import { WalletDTO } from "../dto/wallet.dto";
+import { BalanceDTO } from "../dto/balance.dto";
+import { ChangeWalletNameRequestDTO } from "./dto/change-wallet-name-request.dto";
+import { SendCoinRequestDTO } from "./dto/send-coin-request.dto";
+import { TransactionDTO } from "../dto/transaction.dto";
+import BN from "bn.js";
+import { CreateTransactionRequestDTO } from "./dto/create-transaction-reqeust.dto";
+import { CreateFlushRequestDTO } from "./dto/create-flush-request.dto";
+import { PaginationDTO } from "../dto/pagination.dto";
+import { DepositAddressDTO } from "../dto/deposit-address.dto";
+import { CreateDepositAddressRequestDTO } from "./dto/create-deposit-address-request.dto";
+import {
+  EthWallet,
+  UserWalletPaginationOptions,
+} from "@haechi-labs/henesis-wallet-core/lib/eth/wallet";
 
 @Injectable()
-export class WalletsService {}
+export class WalletsService {
+  public async getWallets(sdk: SDK, name?: string): Promise<WalletDTO[]> {
+    const options = {};
+    if (name != null) {
+      options["name"] = name;
+    }
+    return (await sdk.eth.wallets.getWallets(options)).map(
```

Comment:
이거 name 파라미터 없을 땐 options를 null로 보내야 sdk쪽에서 query string 안만들고 보내지 않나여??

Reply:
```
curl --location --request GET 'localhost:3000/api/v3/ethereum/wallets/' \
--header 'x-henesis-secret: f2sGpgMbhBavVihRyrFp4trhHP7RdOplYDhxY3pGn+c=' \
--header 'Authorization: Bearer eyJhbGciOiJIUzUxMiJ9.eyJlbWFpbCI6InByb2R1Y3RAaGFlY2hpLmlvIiwiaWQiOiI0OTgwM2ZiZjBhMmEyYzNhYjNhMGFiYThiOThkYmIyYSIsInR5cGUiOiJMT05HIiwibG9uZ1R5cGUiOnRydWUsImlzcyI6ImhlbmVzaXMtd2FsbGV0LWRldiIsImlhdCI6MTYwNDkyNDc4MCwiZXhwIjoxNjM2NDYwNzgwfQ.Ks89WsyaKajh661wiMumG7ravb3wtG-gsNpj7e6QQctjTYYiHC7YQa6lx9pF8TMqjc8u2sWpPNSwjH-4H_zV8A' \
--data-raw ''
```

이렇게 요청했는데 

```
  "request": {
    "url": "https://dev.wallet.henesis.io/api/v2/eth/wallets",
    "method": "get",
    "data": null,
    "headers": {
      "Accept": "application/json, text/plain, */*",
      "If-Modified-Since": "Mon, 26 Jul 1997 05:00:00 GMT",
      "Authorization": "Bearer eyJhbGciOiJIUzUxMiJ9.eyJlbWFpbCI6InByb2R1Y3RAaGFlY2hpLmlvIiwiaWQiOiI0OTgwM2ZiZjBhMmEyYzNhYjNhMGFiYThiOThkYmIyYSIsInR5cGUiOiJMT05HIiwibG9uZ1R5cGUiOnRydWUsImlzcyI6ImhlbmVzaXMtd2FsbGV0LWRldiIsImlhdCI6MTYwNDkyNDc4MCwiZXhwIjoxNjM2NDYwNzgwfQ.Ks89WsyaKajh661wiMumG7ravb3wtG-gsNpj7e6QQctjTYYiHC7YQa6lx9pF8TMqjc8u2sWpPNSwjH-4H_zV8A",
      "X-Henesis-Signature": "N3eKYhUzQN2XQ+9jJCoTeHi8Lc2d7S7JRFiee5fXjMA=",
      "X-Henesis-Timestamp": 1619704827385,
      "X-Henesis-User-Agent": "HenesisWallet/2.10.14-alpha.11",
      "User-Agent": "axios/0.21.1"
    },
    "params": null
  },
```

이렇게 요청 날아갔습니다! 즉 이렇게 구현해도 쿼리스트링 안만들어집니다.

Reply:
아하 굿! 감사합니다!!